website/content/docs/secrets/databases/index.mdx

```diff
@@ -148,6 +148,7 @@ and private key pair to authenticate.
 | [PostgreSQL](/docs/secrets/databases/postgresql)      | Yes                      | Yes           | Yes          | Yes (1.7+)             | password                  |
 | [Redshift](/docs/secrets/databases/redshift)          | Yes                      | Yes           | Yes          | Yes (1.8+)             | password                  |
 | [Snowflake](/docs/secrets/databases/snowflake)        | Yes                      | Yes           | Yes          | Yes (1.8+)             | password, rsa_private_key |
+| [Redis](/docs/secrets/databases/redis)                | Yes                      | Yes           | Yes          | No                     | password                  |
```

Comment:
I think it's in alphabetical order so Redis would be between Postgre & Redshift

Reply:
👍 Will update